vtm.js

```diff
@@ -1,11 +1,73 @@
+const DEFAULTNAME = "A Kindred That Is Not To Be Named";
 const app = Vue.createApp({
   data() {
     return {
+      biography: {
+        name: "",
+        defaultName: DEFAULTNAME,/*make a set of names and randomly pick a default on page load*/
+        age: null, /*why not age=0? bcuz if you use a value then placeholder does not apply in input box*/
+        clan: null,
+        generation: null,
+        faction: null,
+      },
+      skillDistributions:[
+        { 
+          id: "Balanced",
+          resource: [12,7,5,3,0,0]
+        },
+        { 
+          id: "Jack-of-all-trades",
+          resource: [8,10,8,1,0,0]
+        },
+        { 
+          id: "Specialist",
+          resource: [17,3,3,3,1,0]
+        },
+      ],
+      clans :[
+        {
+          id: "Thin-blood",
+          abilities: null,
+          bloodPotency: "=0",
```

Comment:
ak je to boolean hodnota, tak by mozno stacilo True/False

Reply:
mozno by stacilo aby to bolo len u Caitiffa a nikde inde

Reply:
ja som si hovoril ze by to mala byt podmienka, mozno budes chciet urobit homebrew kde budu nejake klany potrebovat bp>2 alebo nieco take... blood potency neni domyslena, ja som tam len hodil data, a podmienka pre thin-blood ze mas bp=0 a vseci ostatny aspon 1, este by sa dalo uvazovat ze thin-blood neni upir

Reply:
imho je lepsie mat niekde jednu podmienku nez u dat kazdeho klanu dalsiu polozku a malo by sa to urobit inac

Reply:
vymysli ako...
ja som teraz napr uvazoval o tom, ze keby sa kategorie dali k statom, tak by ta struktura potom bola stejna ako u disciplin a nemusel by sa tam riesit technicky duplicitny algoritmus
samozrejme asi to neni dobry napad zo stejnych dovodov vyssie

Reply:
if clan == Thin Blood ? 
if clan == Caitiff ?

Reply:
imho blood potency nepotrebujeme, aby bolo mozne generovat postavu

Reply:
no preto by som to riesil v inom PR, ale nechce sa mi mazat ten riadok, sice neprinasa uzitok, ale obsahuje logicku vec ktora nejak popisuje problem

Reply:
ktory nepotrebujeme riesit

Reply:
a potom neskor vytvarat PR aby sa to odobralo

Reply:
nie odobralo, ale vyriesilo, prerobilo tak aby to fungovalo

Reply:
tak ked to nefunguje, tak to tam nedavaj. Ja stale nechapem preco na tom trvas, ze to tam chces, ked to nic nerobi. Z pohladu aktualneho produktu, je to uplne zbytocny kod, ktory stazuje citanie. Je to ako zbytocny komentar

Reply:
ja neviem, lebo je to v pravidlach, tak aby som na to nemusel mysliet a nezabudol na to? A lebo kvoli tomu nemozes nastavit default  clan?